src/Program.cs

```diff
@@ -2,7 +2,9 @@
 // Licensed under the MIT License.
 
 using System.CommandLine.Builder;
```

Comment:
Please coordinate with Patrick on his Area PR.  You might want to wait for his - you will be in merge nightmare.